src/libraries/System.Private.CoreLib/src/System/Collections/Generic/EqualityComparer.cs

```diff
@@ -32,6 +32,32 @@ bool IEqualityComparer.Equals(object? x, object? y)
             ThrowHelper.ThrowArgumentException(ExceptionResource.Argument_InvalidArgumentForComparison);
             return false;
         }
+
+        internal virtual int IndexOf(T[] array, T value, int startIndex, int count)
```

Comment:
Could you please add this under `#if !CORERT` to match what is done to the callsite in Array.cs?

Reply:
Done?